src/catalog/src/error.rs

```diff
@@ -82,6 +82,12 @@ pub enum Error {
         backtrace: Backtrace,
     },
 
+    #[snafu(display("Failed to get engine: {}", source))]
+    TableEngineNotFound {
```

Comment:
Please add the engine name to the error msg. And change the message into `Table engine not found: {name}`.